pkgs/top-level/all-packages.nix

```diff
@@ -3242,6 +3242,8 @@ in
     stdenv = gccStdenv;
   };
 
+  fluidasserts = with python37Packages; toPythonApplication fluidasserts;
```

Comment:
you should never use toPythonApplication in `python-modules` doesn't allow for re-use of the python module

Reply:
I think I'm doing it exactly as the documentation says:

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#topythonapplication-function

put the library in python-modules, use toPythonApp in all-packages

what's the correct way to add something that is both app and library ?

Reply:
yea, I think I had it backwards... toPythonApplication should be used in all-packages